common/src/main/java/earth/terrarium/adastra/common/compat/rei/displays/CompressingDisplay.java

```diff
@@ -10,6 +10,7 @@
 
 import java.util.List;
 
+@REIPluginCompatIgnore
```

Comment:
This seems to be missing an import?

`import me.shedaniel.rei.plugincompatibilities.api.REIPluginCompatIgnore;`